index.html

```diff
@@ -5,10 +5,209 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Aruba Ruby Conference 2023</title>
-    <link rel="stylesheet" href="/assets/stylesheets/index.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./assets/stylesheets/app.css" />
+    <link rel="stylesheet" href="./assets/stylesheets/index.css" />
   </head>
   <body>
-    <h1>Hello World!</h1>
-    <script src="/assets/javascript/app.js"></script>
+    <header>
+      <div id="nav-banner">
+        <a
+          href="https://www.facebook.com/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+        <a
+          href="https://www.twitter.com"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#">
+          <span>English</span>
+        </a>
+        <a href="#">
+          <span>My Page</span>
+        </a>
+        <a href="#">
+          <span>Logout</span>
+        </a>
+      </div>
+      <nav id="navbar">
+        <i id="menu-btn" class="fa-solid fa-bars"></i>
+        <a href="./" id="logo">
+          <img src="./assets/images/logo.png" alt="logo" />
+        </a>
+        <ul>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li id="cc_campaign"><a href="#">RoR Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section>
+      <div id="mobile-menu" class="d-none flex-column">
```

Comment:
[OPTIONAL] Kindly fix your mobile as seen in the design provided. It needs to be transparent. And also there is a bug when you the page to the bottom. Please check the gif below.
![Screen Recording 2023-01-06 at 8 21 13 PM](https://user-images.githubusercontent.com/25887253/211066250-ad7a5da7-163f-4eb0-a1d0-39201b697f8b.gif)

Cindy's design:
<img width="822" alt="Screenshot 2023-01-06 at 8 24 17 PM" src="https://user-images.githubusercontent.com/25887253/211066334-57425f2d-49e1-4c32-8b0f-7c7308530f14.png">